docker-image/v1.16/debian-azureblob/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# AUTOMATICALLY GENERATED
+# DO NOT EDIT THIS FILE DIRECTLY, USE /templates/Dockerfile.erb
+
+FROM fluent/fluentd:v1.16.1-debian-amd64-1.2
```

Comment:
Missing image.

```suggestion
FROM fluent/fluentd:v1.16.1-debian-amd64-1.0
```

Reply:
templates/Dockerfile.erb is not touched.

Reply:
Done.